vcd/resource_vcd_vapp_vm.go

```diff
@@ -247,6 +254,7 @@ func vmSchemaFunc(vmType typeOfVm) map[string]*schema.Schema {
 			Type:        schema.TypeString,
 			Optional:    true,
 			Description: "Media name to add as boot image.",
+			//RequiredWith: []string{"boot_image", "catalog_name"},
```

Comment:
Is this a leftover or is it some sort of "guidance" comment?

Reply:
This was me trying to add some validation initially, but then I decided to avoid such changes for this PR.